apigw-lambda-sfn-transcribe-translate-polly-sam/src/invokeTranslator.js

```diff
@@ -1,7 +1,8 @@
-const AWS = require('aws-sdk');
-const stepfunctions = new AWS.StepFunctions();
-const apig = new AWS.ApiGatewayManagementApi({
-    endpoint: process.env.APIG_ENDPOINT,
+const { SFNClient, StartSyncExecutionCommand } = require('@aws-sdk/client-sfn');
+const { ApiGatewayManagementApiClient, PostToConnectionCommand } = require('@aws-sdk/client-apigatewaymanagementapi');
+const stepfunctions = new SFNClient({});
+const apig = new ApiGatewayManagementApiClient({
+    endpoint: `https://${process.env.APIG_ENDPOINT}`,
```

Comment:
note: Some error occurs. Because we need use `connections` url with https.
- https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-how-to-call-websocket-api-connections.html

>< {"message": "Internal server error", "connectionId":"R-c1lc6StjMCFug=", "requestId":"R-d65GZntjMECaQ="}